system/assets/hypha-image-uploader.js

```diff
@@ -0,0 +1,55 @@
+$(function() {
+	$('[data-image-uploader]').each(function () {
+		// get input
+		var $input = $(this);
+		console.log($input.data('image-uploader-src'));
+
+		var src = $input.data('image-uploader-src');
+
+		// Add preview
+		var $preview;
+
+		$input.on('change', function () {
+			createPreviewImage(window.URL.createObjectURL(this.files[0]));
+			toggleInputAndDelete();
+		});
+		var createPreviewImage = function (src) {
+			if (src) {
+				$preview = $('<img class="preview_uploaded_image" src="' + src + '">').insertAfter($input);
+				$('<a href="#"><img src="https://image.flaticon.com/icons/svg/261/261935.svg" style="width:40px;height:40px;" accept="image/png,image/jpeg,image/jpg,image/gif,image/bmp"></a>')
+					.on('click', function (e) {
+						e.preventDefault();
+						// call delete function
+						if ($input.prop('defaultValue') == $preview.attr('src')) {
+							$.post('delete_uploaded_image', {src: $preview.attr('src')});
+						}
+
+						// remove delete button
+						$(this).remove();
+
+						// set image src to null
+						$preview.attr('src', null);
```

Comment:
Why bother with this when you're about to remove the preview entirely?

Reply:
Seems logical, but actually this is needed to work properly.

Reply:
found a way to fix it